partials/footer.php

```diff
@@ -1,20 +1,10 @@
-
-<?php
-  $logo = IGV_get_option('_igv_site_options', '_igv_site_logo');
-?>
-  <footer id="footer" class="container">
-    <div class="grid-row">
-<?php
-  if (!empty($logo)) {
-?>
-      <div class="grid-item item-s-6">
-        <img src="<?php echo esc_url($logo); ?>" id="site-logo">
-      </div>
-<?php
-  }
-?>
-      <div class="grid-item item-s-6 font-family-heading font-lighter font-size-larger">
-        COPYRIGHT <?php echo date('Y'); ?>
-      </div>
+<footer id="footer" class="container margin-top-basic margin-bottom-basic">
+  <div class="grid-row justify-between align-items-center">
+    <div class="grid-item">
+      <?php get_template_part('partials/durango-logo'); ?>
```

Comment:
we've had trouble with file_get_contents in the past because of the home_url / site_url discrepancy. this solves that. i don't see the issue.